src/java.base/share/classes/java/lang/reflect/AccessibleObject.java

```diff
@@ -385,11 +387,15 @@ private void throwInaccessibleObjectException(Class<?> caller, Class<?> declarin
         msg += " " + pn + "\"" ;
         if (caller != null)
             msg += " to " + caller.getModule();
+        throw newInaccessibleObjectException(msg);
+    }
+
+    InaccessibleObjectException newInaccessibleObjectException(String msg) {
```

Comment:
This internal helper method can be `static`:
```suggestion
    static InaccessibleObjectException newInaccessibleObjectException(String msg) {
```